airbyte-integrations/connectors/source-s3/acceptance-test-config.yml

```diff
@@ -88,10 +88,20 @@ acceptance_tests:
     - config_path: secrets/jsonl_newlines_config.json
       # v4
     - config_path: secrets/v4_config.json
+      backward_compatibility_tests_config:
+        disable_for_version: 3.1.2 # This backwards compatibility test cannot pass because it tries to run the v4 config with the v3.1.2 connector
```

Comment:
these backward compatibility tests cannot pass because they run the previous connector on the new configs